internal/commands/sbommonitor/sbommonitor.go

```diff
@@ -26,6 +30,33 @@ func RegisterWorkflows(e workflow.Engine) error {
 	return nil
 }
 
+func loadPolicyFile(policyPath, sbomFilePath string) ([]byte, error) {
+	policy := []byte("\n")
+	var policyFilePath string
+	if policyPath != "" {
+		policyFilePath = policyPath
+	} else {
+		policyFilePath = filepath.Join(filepath.Dir(sbomFilePath), ".snyk")
+	}
+
+	_, err := os.Stat(policyFilePath)
+	if err != nil {
+		if errors.Is(err, fs.ErrNotExist) {
+			return policy, nil
+		} else {
+			return nil, err
+		}
+	}
+
+	bts, err := os.ReadFile(policyFilePath)
+	if err != nil {
+		return nil, err
+	}
+	policy = bts
```

Comment:
i think this should also work:
```suggestion
	policy, err = os.ReadFile(policyFilePath)
	if err != nil {
		return nil, err
	}
```